eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -87,15 +87,15 @@ jobs:
         inputs:
           versionSpec: '14.x'
         displayName: 'Install NodeJS'
+      - template: /eng/pipelines/templates/steps/install-dotnet.yml
       - task: PowerShell@2
         displayName: "Verify generated code"
         env:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
-          DOTNET_MULTILEVEL_LOOKUP: 0
         inputs:
           filePath: "eng/scripts/CodeChecks.ps1"
-          arguments: -ServiceDirectory ${{parameters.ServiceDirectory}}
+          arguments: -ServiceDirectory ${{parameters.ServiceToTest}}
```

Comment:
Do you really want to switch this to verify everything in the core pipeline?

Reply:
Yes, because only the core pipeline is started when the generator/snippet updater/targes are updated and we would like to make sure everything is up to date.